src/ide/code_lens/executables.rs

```diff
@@ -0,0 +1,178 @@
+use crate::config::Config;
+use crate::ide::code_lens::{
+    AnnotatedNode, CodeLensProvider, FileCodeLens, LSCodeLens, LensOwner, PLAIN_EXECUTABLES,
+    collect_functions_with_attrs, get_original_node_and_file,
+};
+use crate::lang::db::AnalysisDatabase;
+use crate::lang::lsp::LsProtoGroup;
+use crate::lang::lsp::ToLsp;
+use crate::lsp::ext::{ExecuteInTerminal, ExecuteInTerminalParams};
+use crate::server::client::Notifier;
+use crate::state::State;
+use cairo_lang_defs::db::DefsGroup;
+use cairo_lang_defs::ids::ModuleId;
+use cairo_lang_executable::plugin::EXECUTABLE_ATTR;
+use cairo_lang_syntax::node::TypedSyntaxNode;
+use cairo_lang_syntax::node::helpers::QueryAttrs;
+use cairo_lang_syntax::node::ids::SyntaxStablePtrId;
+use lsp_types::{CodeLens, Command, Position, Range, Url};
+use scarb_metadata::CompilationUnitMetadata;
+use serde_json::Value;
+use std::collections::HashMap;
+
+#[derive(PartialEq)]
+struct ExecutableLensPayload {
+    command: String,
+}
+
+#[derive(PartialEq)]
+pub struct ExecutableCodeLens {
+    lens: CodeLens,
+    payload: ExecutableLensPayload,
```

Comment:
```suggestion
    command: String,
```
You don't use this field anywhere else anyways